nocom/src/Account.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import './InputBox.css';
+
+function InputBox({ className, onClick }) {
+  const defaultProps = {
+    image: 'https://assets.api.uizard.io/api/cdn/stream/9b130954-67eb-43a8-bd9b-2ef2890b017f.png',
+  };
+
```

Comment:
function Onclick(){
}

Reply:
try to always leave some context in english, even when suggesting code changes @antAmaine